README.md

```diff
@@ -11,12 +11,11 @@ Collection of some open-source services for the osparc simcore platform:
-|  [3d-viewer](services/dy-3dvis/docker/custom/Dockerfile)                                                        |  Paraview Web-based Visualizer                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |  dynamic        |  [![](https://images.microbadger.com/badges/version/itisfoundation/3d-viewer:3.0.3.svg)](https://microbadger.com/images/itisfoundation/3d-viewer:3.0.3 'See Image Version')                                                                                    |                                                                                                                                       |
-|  [3d-viewer-gpu](services/dy-3dvis/docker/custom/Dockerfile)                                                    |  Paraview Web-based Visualizer                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |  dynamic        |  [![](https://images.microbadger.com/badges/version/itisfoundation/3d-viewer-gpu:3.0.3.svg)](https://microbadger.com/images/itisfoundation/3d-viewer-gpu:3.0.3 'See Image Version')                                                                            |                                                                                                                                       |
```

Comment:
hmm is it correct that these 3d-viewer disappears?

Reply:
Mmh, nope. When doing `make toc` it gives this error:
```
Could not decode: [ {"name": "ports", "type": "int", "value": ${SERVER_PORT:-80}}, {"name": "constraints", "type": "string", "value": ["node.platform.os == linux"]}, {"name": "resources", "type": "resources", "value": { "Limits": { "NanoCPUs": 4000000000, "MemoryBytes": 17179869184 }, "Reservations": { "NanoCPUs": 100000000, "MemoryBytes": 536870912 } } } ] for 3d-viewer
Expecting value: line 1 column 45 (char 44)
Could not decode: [ {"name": "ports", "type": "int", "value": ${SERVER_PORT:-80}}, {"name": "constraints", "type": "string", "value": [ "node.platform.os == linux" ]}, {"name": "Resources", "type": "Resources", "value": { "Limits": { "NanoCPUs": 4000000000, "MemoryBytes": 17179869184 }, "Reservations": { "NanoCPUs": 100000000, "MemoryBytes": 536870912, "GenericResources": [ { "DiscreteResourceSpec": { "Kind": "VRAM", "Value": 1 } } ] } } } ] for 3d-viewer-gpu
Expecting value: line 1 column 45 (char 44)
```

I think the problem comes from the [docker-compose-meta.yml](https://github.com/elisabettai/osparc-services/blob/master/services/dy-3dvis/docker-compose-meta.yml#L29) for those services. 

To it removes them from the table, I guess...

Reply:
@sanderegg, so the problem was with the "$" in the [metadata settings for the 3d-viewers](https://github.com/elisabettai/osparc-services/blob/master/services/dy-3dvis/docker-compose-meta.yml#L61).
They generated the error above and so they were skipped in the table.

It seems that if I use [here](https://github.com/ITISFoundation/osparc-services/blob/master/scripts/auto-doc/create-toc.py#L109):
`settings_list = json.loads(json.dumps(service["build"]["labels"]["simcore.service.settings"]))`
instead of:  
`settings_list = json.loads(service["build"]["labels"]["simcore.service.settings"].encode("utf8"))`

fixes the issue. Let me know if this doesn't make sense.